apps/webcert/src/store/certificate/certificateMiddleware.ts

```diff
@@ -241,12 +241,12 @@ const handleDeleteCertificateSuccess: Middleware<Dispatch> =
   ({ dispatch }) =>
   () =>
   (action: AnyAction): void => {
+    dispatch(hideSpinner())
```

Comment:
Does it make more sense to move this dispatch to line 252, and hide the spinner after the other dispatches has finished?

Reply:
I don't think so, at least not in any meaningful way, it would be milliseconds for every dispatch to fire and update state